src/Symfony/Component/Console/Completion/CompletionSuggestions.php

```diff
@@ -26,25 +26,29 @@ class CompletionSuggestions
     /**
      * Add a suggested value for an input option or argument.
      *
+     * @param string|Suggestion $value
+     *
      * @return $this
      */
-    public function suggestValue(string $value): self
+    public function suggestValue($value): self
     {
-        $this->valueSuggestions[] = $value;
+        $this->valueSuggestions[] = !$value instanceof Suggestion ? new Suggestion($value) : false;
```

Comment:
passing a suggestion should store it instead of storing `false`

Reply:
We need to add a test case with `$value instanceof Suggestion`.